src/fprime/cookiecutter_templates/cookiecutter-fprime-project/{{cookiecutter.project_name}}/project.cmake

```diff
@@ -1,2 +1,4 @@
-# This CMake file is intended to register project-wide objects so they can be
-# reused easily between deployments, but also by other projects.
+# This CMake file is intended to register project-wide objects.
+# This allows for reuse between deployments, or other projects.
+
+add_fprime_subdirectory("${CMAKE_CURRENT_LIST_DIR}/Components")
```

Comment:
This seems to require `Components` as opposed to just using it when the user creates this.  Intended?

Reply:
Yes, I have added the `Components/` module as part of the project generation. I can revert back, but ...

>I also took the liberty to add the Components/ module to the project generation tree because it sounds to me like a good pattern to suggest to users - while it does not force anything onto them.

Reply:
It is fine!